awx/ui_next/src/screens/Template/Templates.jsx

```diff
@@ -52,6 +52,13 @@ class Templates extends Component {
       [`/templates/${template.type}/${template.id}/completed_jobs`]: i18n._(
         t`Completed Jobs`
       ),
+      [`/templates/${template.type}/${template.id}/schedules`]: i18n._(
+        t`Schedules`
+      ),
+      [`/templates/${template.type}/${template.id}/schedules/${nestedResource &&
```

Comment:
could we rename nestedResource to schedule inside this function

Reply:
Oh yea, good idea - I just copied it over from templates